core/storage/suggestion/gae_models.py

```diff
@@ -0,0 +1,81 @@
+# coding: utf-8
```

Comment:
I agree. Suggestions are a subset of feedback threads, so the suggestions model classes should go within feedback's gae_models file.